tests/raftstore/server.rs

```diff
@@ -112,16 +116,16 @@ impl Simulator for ServerCluster {
         cfg.addr = format!("{}", addr);
 
         let simulate_trans = SimulateTransport::new(strategy, trans.clone());
-        let mut node = Node::new(&cfg,
-                                 self.pd_client.clone(),
-                                 Arc::new(RwLock::new(simulate_trans)));
+        let trans_final = Arc::new(RwLock::new(simulate_trans));
```

Comment:
the actual Transport we use...
I save a copy in ServerCluster simulate_trans field for later use (i.e. change filters of it to hook transport)